src/views/AnalysisFramework/index.js

```diff
@@ -209,7 +153,9 @@ export default class AnalysisFramework extends React.PureComponent {
             projectId,
         } = this.props;
 
-        if (!analysisFramework) {
+        const { pendingFramework } = this.state;
+
+        if (!analysisFramework || pendingFramework) {
```

Comment:
analysisFramework is set to {} in selector so !analysisFramework is always false.
Anyway, analysisFramework should not be used to identify pending state.